examples/solids/README.rst

```diff
@@ -68,7 +69,13 @@ With the sidesets defined in the figure, we provide here an example of a minimal
 
 In this example, we set the left boundary, face set :math:`999`, to zero displacement and the right boundary, face set :math:`998`, to displace :math:`0` in the :math:`x` direction, :math:`-0.5` in the :math:`y`, and :math:`1` in the :math:`z`.
 
-As an alternative to specifying a mesh with :code:`-mesh`, the user may use a DMPlex box mesh by specifying :code:`-dm_plex_box_faces [int list]`, :code:`-dm_plex_box_upper [real list]`, and :code:`-dm_plex_box_lower [real list]`.
+As an alternative to specifying a mesh with :code:`-mesh`, the user may use a DMPlex box mesh by specifying :code:`-dm_plex_box_faces [int list]`, :code:`-dm_plex_box_upper [real list]`, and :code:`-dm_plex_box_lower [real list]`. 
```

Comment:
Not sure what changed in this sentence?

Reply:
I copied and pasted  `:code:` from there as I keep typing a wrong thing and I guess `make style` thought something changed there!!!